src/locales/en.json

```diff
@@ -63,24 +63,19 @@
     "step accounts next": "Browse",
     "myaccounts info": "You can now access your Cozy at anytime by clicking 'Continue'",
     "myaccounts next": "Continue",
+    "sign in": "Sign in",
```

Comment:
I think that the local key should be more contextualized, to make it clear that it concerns connexion to konnector services. That was it was previously something like `intent service connect`. What do you think about `konnector sign in` or `konnector service sign in` ?